src/Fixer/Operator/NewWithBracesFixer.php

```diff
@@ -98,6 +98,10 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
                 [CT::T_BRACE_CLASS_INSTANTIATION_OPEN],
                 [CT::T_BRACE_CLASS_INSTANTIATION_CLOSE],
             ];
+            if (\defined('T_AMPERSAND_FOLLOWED_BY_VAR_OR_VARARG')) { // @TODO: drop condition when PHP 8.1+ is required
```

Comment:
not needed, this is used in `getNextTokenOfKind` which uses `equalsAny` which uses `equals` which you fixed to deal with `&` itself

Reply:
Seems we need it, when I remove it some tests are failing.

Call for `getNextTokenOfKind`, then `getTokenOfKindSibling` is doing tokens filtering with `isTokenKindFound` before it is using `equals` method.